### ISSUE TITLE
diskrec windows not reporting all disks

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#83]
It has been observed on certain systems we dont correctly report disk throghput and stats if we have a single physical drive, 2 logical volumes on it:

```
C:\Users\SMA>"C:\Program Files\Kronometrix\bin\disks.exe"
\\\\.\\PHYSICALDRIVE0 Disk #0, Partition #1, Logical: C:
\\\\.\\PHYSICALDRIVE0 Disk #0, Partition #1, Logical: D: 
```

Kronometrix shows only disk0-d not c. Why ? Bugs, bugs ...

**Comment by sparvu** [comment]
A probe into such system, shows diskrec.exe reporting a single drive, disk0_d where disk0_C is skipped all together.

```
c:\Program Files\Kronometrix\bin>diskrec.exe -v
info: last_time=0
info: Number of disks: 1
info: disk0 => D:
info: elapsed=0.499206(ms), sleep=0.500794(ms), timenow=1452177299(epoch)
1452177299:disk0_d:0:0.00:0:0:0.00:0:0:0:0:0:0.00:71.49:69.08:4
```

**Comment by sparvu** [comment]
More details here:

```
$ ./diskrec2.exe
disk: 0, C:
disk: 0, D:
1452190287:disk0_d:0:0.00:0:0:0.00:0:0:0:0:0:0.00:71.49:68.98:4
```

So it seems the disks are correctly found, but only for one we get stats. This means:

 * getdisk() is correct. Does return all disks volumes attached to the system

 * something is broken on the stat procedure

**Comment by sparvu** [comment]
fixed in 1.0.18 RC2:

```
$ ./diskrec2.exe
1452242013:disk0c:0:0.00:0:0:0.00:0:0:0:0:0:0.00:380.00:325.54:15
1452242013:disk0d:0:0.00:0:0:0.00:0:0:0:0:0:0.00:71.49:68.99:4
```

I have replaced as well the device id in a simpler form: disk0c instead of disk0_c ...

**Comment by sparvu** [comment]
fixed in RC2.
